e2e-tests/tfm/cypress/fixtures/deal-AIN.js

```diff
@@ -190,6 +190,7 @@ const MOCK_DEAL = {
       currencySameAsSupplyContractCurrency: 'true',
       riskMarginFee: '10',
       coveredPercentage: '20',
+      coverStartDate: `${dateConstants.twoYearsAgoUnix}000`,
```

Comment:
This is the facility that's used in all the e2e amendments tests, I'm not sure why coverStartDate was omitted as a field here before, but I need it for the:
 `it('should return errors when a date before the cover start date is entered', () => {`  facility end date e2e test so that the entered date has a cover start date to be compared against. Agreed on the existing pattern being grim